kafka-client/src/main/java/dev/responsive/kafka/store/SizeTrackingBuffer.java

```diff
@@ -41,6 +41,8 @@ public NavigableMap<K, Result<K>> getReader() {
   }
 
   private long sizeOf(final K key, final Result<K> value) {
-    return extractor.bytes(key).get().length + (value.isTombstone ? 0 : value.value.length);
+    return extractor.bytes(key).get().length
+        + (value.isTombstone ? 0 : value.value.length)
+        + Long.BYTES; // timestamp size
```

Comment:
nit: can we move this method to the `Result` class? Feels way too easy to miss updating the size computation if/when the format changes